<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loteria</class>
 <widget class="QMainWindow" name="loteria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MEGA-SENHA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-style:outset;
border-radius:5px;
border-width:0px;
background-color: rgb(100, 174, 16);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>641</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Trebuchet MS;
font-size: 15px;
font-weight: bold;
color: white;
border:none;
text-decoration:underline;
color:white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,0,0), stop:1 rgba(1, 181, 180));
</string>
    </property>
    <property name="text">
     <string>SEJA BEM VINDO MEGA SENA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="roda_roda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: white;
border:none;
color:white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,0,0), stop:1 rgba(1, 181, 180));
</string>
    </property>
    <property name="text">
     <string>MEGA SENA</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="numero1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:15px;
border-width:0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sistema/icone py/sinal-de-visto.png</normaloff>sistema/icone py/sinal-de-visto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="numero2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:15px;
border-width:0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sistema/icone py/sinal-de-visto.png</normaloff>sistema/icone py/sinal-de-visto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="numero3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:15px;
border-width:0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sistema/icone py/sinal-de-visto.png</normaloff>sistema/icone py/sinal-de-visto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="numero4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:15px;
border-width:0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sistema/icone py/sinal-de-visto.png</normaloff>sistema/icone py/sinal-de-visto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="numero5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:15px;
border-width:0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sistema/icone py/sinal-de-visto.png</normaloff>sistema/icone py/sinal-de-visto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="numero6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:15px;
border-width:0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>sistema/icone py/sinal-de-visto.png</normaloff>sistema/icone py/sinal-de-visto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="limpar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-radius:8px;
border-width:0px;
background-color: rgba(1,181, 180);
color:white;</string>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>iconloteria/deletar_button.png</normaloff>iconloteria/deletar_button.png</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,0,0), stop:1 rgba(1, 181, 180));

color:white;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::BottomToTop</enum>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_app">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Trebuchet MS;
font-size: 15px;
font-weight: bold;
color: white;
border:none;
text-decoration:underline;
color:white;
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,0,0), stop:1 rgba(1, 181, 180));
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 181, 180);
border-style:outset;
border-radius:40px;
border-width:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>roda_roda</zorder>
   <zorder>label_welcome</zorder>
   <zorder>numero1</zorder>
   <zorder>numero2</zorder>
   <zorder>numero3</zorder>
   <zorder>numero4</zorder>
   <zorder>numero5</zorder>
   <zorder>numero6</zorder>
   <zorder>limpar</zorder>
   <zorder>progressBar</zorder>
   <zorder>text_app</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
